baybe/objectives/base.py

```diff
@@ -14,8 +14,11 @@
 from baybe.serialization.mixin import SerialMixin
 from baybe.targets.base import Target
 
+# TODO: Reactive slots in all classes once cached_property is supported:
+#   https://github.com/python-attrs/attrs/issues/164
 
-@define(frozen=True)
+
+@define(frozen=True, slots=False)
```

Comment:
did this cause an issue with hypothesis or is is just for consistency?

Reply:
No, not with hypothesis but because I added a cached property to the class (had to refactor the weight normalization issue due to the reason I briefly mentioned in one of our meetings, i.e. stuff was renormalized after floating point inaccuracies, which cause the roundtrips to fail. Now we store the raw weights instead and normalize on the fly). Still waiting for the cached property support in slotted attrs classes, which will come in the next release (it's already available on main), so same as #74